### ISSUE TITLE
Fiat + Currency Converter

### ISSUE THREAD

**Issue by Drewkid061** [yours-org/yours-wallet#261]
Hi.

Can you please add the following API. This would enable us to change between various currencies.

https://github.com/fawazahmed0/exchange-api

https://cdn.jsdelivr.net/npm/@fawazahmed0/currency-api@{date}/{apiVersion}/{endpoint}

Thank you.

**Comment by danwag06** [comment]
Happy to review a PR... this would be a nice feature.

**Comment by wheval** [comment]
Hello, Can I work on this?

**Comment by Benjtalkshow** [comment]
> Happy to review a PR... this would be a nice feature.

I would be glad to work on this issue. Can I get started?

**Comment by Michaelkingsdev** [comment]
@danwag06 
I have the skillset to tackle this issue. Can it be assigned to me?

**Comment by 0xdevcollins** [comment]
Can I be assigned this issue?

**Comment by od-hunter** [comment]
Hi, can I be assigned this please? I’m ready to work.

**Comment by michojekunle** [comment]
Can I take this issue?

**Comment by cryptogru725** [comment]
Can I jump on this task?

**Comment by kuopassa** [comment]
^ What's this spam?

**Comment by Emmy123222** [comment]
May I take care of this?

**Comment by Viktohblake** [comment]
Can I jump on this task?

**Comment by chiscookeke11** [comment]
Can I attempt this issue?

**Comment by tebogomap** [comment]
Could I be assigned to this?

**Comment by Drewkid061** [comment]
Hi @danwag06 , will there be any change that this feature might be implemented in the new update, or are you still testing it.

**Comment by kuopassa** [comment]
Looks like supply chain attack to me.

**Comment by Elite-tch** [comment]
I'd like to take this issue.

**Comment by Ekene001** [comment]
Can I take on this task? I’m a full-stack blockchain developer with over 3 years of experience working with Tailwind, React, TypeScript, and Solidity/Rust.

**Comment by chiscookeke11** [comment]
Can I attempt this issue?

**Comment by B-uchi** [comment]
I'd love to work on this!

**Comment by emarc99** [comment]
Is this issue still available? Would love to tackle it.

**Comment by TobieTom** [comment]
Yes I can actually tackle this. I am new in this space too so I am looking to add this to my experience

**Comment by laryhills** [comment]
Hi, I'm Hilary, a full-stack Web Developer and I will like to take on this issue.

ETA - 2 -3 days

**Comment by Verifieddanny** [comment]
Could I take on this issue?

**Comment by viktrified** [comment]
Can I take care of this issue?
